contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -307,6 +309,23 @@ contract CoreIssuanceOrder is
 
         // Make sure fill or cancel quantity is multiple of natural unit
         require(_executeQuantity % set.naturalUnit() == 0, "FILL_NOT_NATURAL_UNIT_MULTIPLE");
+
+        // Make sure required components array is non-empty
+        require(_order.requiredComponents.length > 0, "ORDER_REQ_COMPONENTS_EMPTY");
```

Comment:
We updated our error messages to a new format that these should be updated to as well.